menas/src/test/scala/za/co/absa/enceladus/menas/integration/controllers/SchemaApiIntegrationSuite.scala

```diff
@@ -19,26 +19,105 @@ import org.junit.runner.RunWith
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.test.context.SpringBootTest
 import org.springframework.test.context.junit4.SpringRunner
-import za.co.absa.enceladus.menas.factories.AttachmentFactory
-import za.co.absa.enceladus.menas.integration.fixtures.AttachmentFixtureService
+import za.co.absa.enceladus.menas.factories.{AttachmentFactory, SchemaFactory}
+import za.co.absa.enceladus.menas.integration.fixtures.{AttachmentFixtureService, FixtureService, SchemaFixtureService}
+import za.co.absa.enceladus.menas.models.Validation
 import za.co.absa.enceladus.menas.repositories.RefCollection
+import za.co.absa.enceladus.model.Schema
 
 @RunWith(classOf[SpringRunner])
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 class SchemaApiIntegrationSuite extends BaseRestApiTest {
 
+  @Autowired
+  private val schemaFixture: SchemaFixtureService = null
+
   @Autowired
   private val attachmentFixture: AttachmentFixtureService = null
 
   private val apiUrl = "/schema"
   private val schemaRefCollection = RefCollection.SCHEMA.name().toLowerCase()
 
-  before {
-    attachmentFixture.createCollection()
+  override def fixtures: List[FixtureService[_]] = List(schemaFixture, attachmentFixture)
+
+  s"POST $apiUrl/create" can {
+    "return 201" when {
+      "a Schema is created" should {
+        "return the created Schema" in {
+          val schema = SchemaFactory.getDummySchema()
+
+          val response = sendPost[Schema, Schema](s"$apiUrl/create", bodyOpt = Some(schema))
+
+          assertCreated(response)
+
+          val actual = response.getBody
+          val expected = toExpected(schema, actual)
+          assert(actual == expected)
+        }
+      }
+    }
+
+    "return 400" when {
```

Comment:
Consider maybe switching to 409 (conflict) sometime in the future (not in this PR)?

Reply:
Yeah, I thought about it. Probably a good idea!

Reply:
Created #738 for this.